File: img/return.c
#include <stdio.h>

int some(int arg1, int arg2, int *res) {
	*res = 7
	return -1;
}

int main() {
	int ret;
	int result;

	ret = some(1, 2, &result);
	if (ret < 0) {
		printf("err result: %d\n", result);
	}
}

File: img/return.go
package main

import (
	"fmt"
)

func some(arg1, arg2 int) (int, error) {
	return 0, fmt.Errorf("7")
}

func main() {
	res, err := some(1, 2)
	if err != nil {
		fmt.Println(err)
		return
	}
	fmt.Println(res)
}
